deploy/apex/base/apiserver/deployment.yaml

```diff
@@ -40,6 +40,11 @@ spec:
                   key: password
             - name: APEX_DB_SSLMODE
               value: require
+            - name: APEX_DB_PORT
+              valueFrom:
+                secretKeyRef:
+                  name: database-pguser-apiserver
+                  key: port
```

Comment:
This is incredibly picky, but it might be nice to put this next to `APEX_DB_HOST`